### PR TITLE
feat: landing page and installation

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Landing page and the Installation page.

- Landing Page:
image

- Installation Page: 
image

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Good to merge.
Landing page is looking clean.

File: index.html [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Andika+New+Basic:ital,wght@1,700&family=Comforter&family=Montserrat:ital,wght@1,800&family=Open+Sans:wght@300;500&family=Roboto:wght@100;400;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
 </head>
+
 <body>
-    <h1>Landing Page</h1>
+    <div class="hero documentation-section">
```
Comment (rahulyadav139) [inline index.html]: Use semantic HTML. If hero is defined for header then use header tag, if you are defining for main section then use main html tag. Other than that landing page is looking awesome. Specially "clay" logo.

**Review by rahulyadav139 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (SinchanaMS) [reply index.html]: Thank you for pointing this out, Rahul! 
Corrected it now.

**Comment by SinchanaMS** [comment]
> Good to merge. Landing page is looking clean.

Thank you, Monu!

**Review by kushalsourav — APPROVED** [review]
good to merge,  dont create extra color var for border-bottom like that 
and for hover you can create like --cover-secondary-hover ==> --cover-secondary-light  + dark like that , your choice.

**Review by pankaj-prog — APPROVED** [review]
Everything looks good to me. 
Code is clean, class names are descriptive.
Also you have used the best practices.
Good to merge

**Comment by SinchanaMS** [comment]
Thank you for the reviews. Merging to dev.

**Pull request merged**